.gitignore

```diff
@@ -12,6 +12,8 @@ hip_build/
 OUTPUT_IMAGES*
 OUTPUT_PERFORMANCE*
 QA_RESULTS*
+OUTPUT_AUDIO*
+QA_AUDIO*
```

Comment:
This also need not be added, i feel
we can have one folder QA_RESULTS which can be used for both image and audio

Reply:
Done